### PR TITLE
Add list of providers of commercial support

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -30,6 +30,15 @@
     more elaborate or controversial changes, please discuss them on the mailing
     list before sending a pull request.
   </p>
+  <h1>Commercial</h1>
```
Comment (juliusv) [inline content/community.html]: s/Commercial/Commercial support/ (caps don't matter, as it'll be all-caps anyways)

File: content/community.html [hunk 2]
```diff
@@ -30,6 +30,15 @@
...
+  <p>
+    This is a list companies which provide supporting or consulting services for
+    Prometheus. Prometheus is an independent open source project which does not
+    endorse any company.
```
Comment (juliusv) [inline content/community.html]: missing "of". Could you also change "companies" -> "third-party companies" to add even more emphasis on this fact?

File: content/community.html (hunk 1 above)
Comment (brian-brazil) [inline content/community.html]: Done

File: content/community.html (hunk 2 above)
Comment (brian-brazil) [inline content/community.html]: Done

**Comment by juliusv** [comment]
I don't see the changes on GitHub...

File: content/community.html [hunk 3]
```diff
@@ -30,6 +30,15 @@
     more elaborate or controversial changes, please discuss them on the mailing
     list before sending a pull request.
   </p>
+  <h1>Commercial support</h1>
+  <p>
+    This is a list of third-party companies which provide supportt or
```
Comment (juliusv) [inline content/community.html]: supportt -> support

File: content/community.html [hunk 4]
```diff
@@ -30,6 +30,15 @@
...
+    consulting services for Prometheus. Prometheus is an independent open source
+    project which does not endorse any company.
+  </p>
+    <ul>
```
Comment (juliusv) [inline content/community.html]: The `<ul>` should either not be indented, or be wrapped in another `<p>` or something.

File: content/community.html (hunk 3 above)
Comment (brian-brazil) [inline content/community.html]: Fixed

File: content/community.html (hunk 4 above)
Comment (brian-brazil) [inline content/community.html]: Fixed. Vim always does odd indenting with html.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**